src/Options.php

```diff
@@ -1150,6 +1191,16 @@ public function getHttpContext()
         return $this->httpContext;
     }
 
+
+    public function validateArtifactPath(?string $path, string $option)
+    {
+        $parsed_uri = parse_url($path);
```

Comment:
This creates a PHP warning: `parse_url(): Passing null to parameter #1 ($url) of type string is deprecated in vendor/dompdf/dompdf/src/Options.php on line 1152`

Reply:
Thank you for that. I really flubbed that implementation 😞.

Reply:
If you find the time it would be nice to have it fixed on 2.0.x :)

Reply:
Absolutely! I will do both.

Reply:
phpunit has a setting to convertdeprecationsToExceptions

Maybe it would work

Reply:
Indeed, it does! Thank you.